addons/website_event_booth/static/src/js/booth_register.js

```diff
@@ -40,20 +40,22 @@ publicWidget.registry.boothRegistration = publicWidget.Widget.extend({
             event_booth_ids: eventBoothIds,
         }).then(function (result) {
             if (result.unavailable_booths.length) {
-                self.$('input[name="event_booth_ids"]').each(function (i, el) {
+                self.el.querySelectorAll("input[name='event_booth_ids']").forEach(function (el) {
```

Comment:
Isn't `for (const el of self.el.querySelectorAll("input[name='event_booth_ids']")) {` lighter and easier to read ?

Reply:
Like it was decided not to alter to alter the code as much as possible, therefore we simply focused on proving the expected alternative for the jquery code. Y